src/views/Home/components/Question/index.js

```diff
@@ -0,0 +1,233 @@
+import React, { Component } from 'react';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import GordonLoader from '../../../../components/Loader';
+import { getQuestions } from './questionData';
+import FormControl from '@material-ui/core/FormControl';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormLabel from '@material-ui/core/FormLabel';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import Radio from '@material-ui/core/Radio';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import Divider from '@material-ui/core/Divider';
+import { Button } from '@material-ui/core';
+import { gordonColors } from '../../../../theme';
+import './index.scss';
+import wellness from '../../../../services/wellness.js';
+
+export default class Question extends Component {
```

Comment:
A top-level comment would be very helpful.

Reply:
added comment /* creates the question for the health check feature */